src/test/java/redis/clients/jedis/tests/utils/AssertUtil.java

```diff
@@ -15,6 +14,15 @@
 
 public class AssertUtil {
 
+  public static boolean assertCollectionContains(Collection<byte[]> array, byte[] expected) {
+    for (byte[] bytes : array) {
+      if (Arrays.equals(bytes, expected)) {
```

Comment:
@sazzad16 shouldn't you use assertArrayEquals here?

Reply:
@gkorland NO. We're checking whether `array` contains `expected`. In case where `expected` is present but not the first element in `array`, assertArrayEquals would throw an assertion error right away, causing the check to fail wrongfully.

Reply:
@gkorland also, check the assertion error (`ComparisonFailure`)  thrown right after the loop.

Reply:
:+1: